### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

**Comment by corymosiman12** [comment]
Aww thanks man

**Comment by nllong** [comment]
> Aww thanks man

I was still a punk and put my name first though... mainly because if someone wants to contact us they would probably start with the first on the list.

File: README.md [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
+The BuildingSync-Gem is a repository of helpers for reading and writing BuildingSync XML files, and for using that data 
+to drive energy simulations of the subject building. See full documentation on [RubyDoc](https://www.rubydoc.info/github/BuildingSync/BuildingSync-gem).
 
 All of the following are supported: 
 
-* convert BuildingSync XML file into: 
+  * convert BuildingSync XML file into: 
+      * an OpenStudio Baseline model 
+      * an OpenStudio workflow for each scenario defined in the XML file 
+  * enables simulation of the baseline model and all workflows and
```
Comment (JieXiong9119) [inline README.md]: enables -> enable or converting/enabling/inserting for each line.
Just trying to be consistent in the tense.

**Review by JieXiong9119 — APPROVED** [review]
Address minor grammar comment and then good to go!

**Pull request merged**